package.json

```diff
@@ -9,11 +9,16 @@
     "format": "prettier --write ."
   },
   "dependencies": {
+    "@emotion/css": "^11.7.1",
```

Comment:
Very small nitpick but I tend to stay away from using the `~` or `^` so that in the case a  future minor/patch version introduces security vulnerabilities or causes and breakages. Not tied to that convention but something I have been wary of.

Reply:
Here is an article about it but I had a better before https://docs.renovatebot.com/dependency-pinning/

Reply:
Again not tied to this approach but was just was something practiced on University and previous projects

Reply:
This is another good article too https://erichurst.dev/when-to-pin-your-projects-dependencies/

Reply:
Ah ok that's interesting, pinning seems like a safe bet. Does Artifcatory kind of handle that though? I guess I'm not sure what the reason for using our own package registry is lol. I assumed the packages were like curated before being added to the registry, or is Artifactory solving some other problem?

Reply:
@mcolella14 - as far as I remember we pointed to our internal registry only when consistent nav came into picture, which was developed by internal design systems team. All the other packages are downloaded from external.